src/sentry/static/sentry/app/components/forms/form.tsx

```diff
@@ -28,11 +28,11 @@ type FormProps = {
   extraButton?: React.ReactNode;
 };
 
-type FormState = {
+type FormClassState = {
```

Comment:
This was being used as both a type and value previously. not great. Now `FormState` is an enum and this type is renamed.